README.md

```diff
@@ -69,6 +69,33 @@ config :airbrake,
   ignore: :all # to disable reporting
 ```
 
+## Shared options for reporting data to Airbrake
+
+To include with every report to Airbrake a set of optional data, include the `:options` key in the config. This can either
+be a keyword list of options or a function that returns a keyword list of options. Keyword list keys that can be used are
+`:context`, `:params`, `:session`, and `:env`.
+
+### Options function in config
+
+A function for creating options for reporting should be declared in the config as a tuple of 
+`{ModuleName, :function_name}`. This function should take as an argument a keyword list, possibly empty and should
+return a keyword list.
+
+```elixir
+config :airbrake,
+  options: {Web, :airbrake_options}
```

Comment:
I’m inclined to add the arity to this, even though it would be hard-coded everywhere to 1.  Just to remain consistent with Elixir itself (see `Function.capture/3`).

Reply:
- Would that give the mistaken impression that a consumer of the library had the option of having a function with arity other than 1?
- How would we use those data in the place where the tuple is consumed?
- Would it be solely for consistency like you described?

The first of those questions is the biggest one in my mind.

Reply:
* I think the usual Elixir thing is that the one specifying "use this function" doesn't have a choice in arity, but we specify it anyway.  We put a `/1` on `&String.downcase/1` when we pass it to `Enum.map/2` even though we all know that `Enum.map/2` takes a function with one argument.
* I had a similar thing with `Token.decode_and_verify/3` in `event_bus_subscribers`.  I pattern matched on the `3`: https://github.com/CityBaseInc/event_bus_subscribers/blob/e03defb7e67b135df8b4efa90051feda74a03d12/lib/event_bus_subscribers/helpers/context.ex#L185.  (If I were nice, I would have done a `case` with a better error message than "failed match".)
* When I saw the tuple, I thought, "Why no arity?  Elixir coders always write an arity?"  So I think it's the "no surprise" aspect of consistency that I'm appealing to here.  I'm pretty sure I'll always have that response when I see this code.

Reply:
(thinking emoji)

Reply:
@jdfrens Please see https://github.com/CityBaseInc/airbrake-elixir/pull/1/commits/604c1ecf1fc81b048c192942396e60ceec2cd8a2